Kernel/Arch/aarch64/ASM_wrapper.h

```diff
@@ -125,6 +125,25 @@ inline u64 read_rndrrs()
     return value;
 }
 
+inline FlatPtr get_cache_line_size()
+{
+    FlatPtr ctr_el0;
+    asm volatile("mrs %[value], ctr_el0"
+                 : [value] "=r"(ctr_el0));
+    auto log2_size = (ctr_el0 >> 16) & 0xF;
```

Comment:
consider moving reading to `Registers.h` if thereʼs more interesting information to be read from that register (only the extraction of relevant raw data; the computations should then be kept here, of course).